<!-- -->
<launch>

  <node name="manual_controller" pkg="robor_controllers" type="manual_controller_node">
    <param name="active"        value="true"/>
    <param name="publish_reference_twist" value="false"/>

    <param name="use_keys"      value="false"/>
    <param name="use_joy"       value="false"/>

    <param name="loop_rate"     value="100.0"/>
    <param name="time_constant" value="0.0"/>

    <param name="linear_gain"   value="0.3"/>
    <param name="angular_gain"  value="0.5"/>
  </node>

  <node name="feedback_controller" pkg="robor_controllers" type="feedback_controller_node">
    <param name="active"       value="true"/>
    <param name="run"          value="false"/>
    <param name="use_ff"       value="true"/>

    <param name="loop_rate"    value="100.0"/>

    <param name="gain_x"       value="0.5"/>
    <param name="gain_y"       value="0.5"/>
    <param name="gain_theta"   value="0.5"/>

    <param name="max_u"        value="0.5"/>
    <param name="max_v"        value="0.5"/>
    <param name="max_w"        value="3.0"/>

    <param name="robot_frame_id" value="robot"/>
    <param name="reference_frame_id" value="reference"/>
  </node>

  <node name="potentials_controller" pkg="robor_controllers" type="potentials_controller_node">
    <param name="active"       value="false"/>
    <param name="run"          value="false"/>
    <param name="assisted_control" value="false"/>

    <param name="loop_rate"    value="100.0"/>

    <param name="R"            value="0.5"/>
    <param name="eta"          value="1.2"/>
    <param name="delta"        value="0.2"/>
    <param name="ko_velocity"  value="0.15"/>

    <param name="gain_pose"    value="0.4"/>
    <param name="gain_theta"   value="3.0"/>

    <param name="max_u"        value="0.4"/>
    <param name="max_v"        value="0.4"/>
    <param name="max_w"        value="3.0"/>

    <param name="fixed_frame_id"     value="map"/>
    <param name="robot_frame_id"     value="robot"/>
    <param name="reference_frame_id" value="reference"/>
  </node>

</launch>

